packages/blockchain-api/src/currencyConversion/index.ts

```diff
@@ -0,0 +1 @@
+export { default as CurrencyConversionAPI } from './CurrencyConversionAPI'
```

Comment:
Why create this index file instead of having other files import from CurrencyConversionAPI directly?

Reply:
Mostly to hide the internal file organisation of currency conversion with it's somewhat private internal module.
But doesn't really matter.